unnamed_toaster/unnamed_toaster_hw_interface/scripts/ros_table_interface.py

```diff
@@ -34,6 +34,7 @@
 from unnamed_toaster_hw_interface.robot_control import RobotControl
 from unnamed_toaster_hw_interface.robot_odom import RobotOdom
 from unnamed_toaster_hw_interface.turret import Turret
+from unnamed_toaster.unnamed_toaster.scripts.match_logic import MatchLogic
```

Comment:
I would suggest we not do this - keep the match logic in a separate node from the hw_interface -- it will allow us to update things much faster (we won't have to restart the entire set of drivers every time we need to change match logic).

Reply:
Thats what I wanted to do too but I need an NT inteface. Should i just create a new instance?

Reply:
How does it look now?